features/step_definitions/capcha_steps.rb

```diff
@@ -3,7 +3,7 @@
 RECAPTCHA_SCRIPT = 'script[src^="https://www.google.com/recaptcha/api.js"]'
 
 Then /^I should not see the captcha$/ do
-  page.should_not have_selector(RECAPTCHA_SCRIPT)
+  page.should_not have_selector(RECAPTCHA_SCRIPT, visible: false)
```

Comment:
This test looks like a "should **not** have an **invisible** RECAPTCHA_SCRIPT selector".
The idea is that we're showing the captcha or not?

Reply:
yes, the step name is confusing: it sounds like it checks for the captcha visibility, but it checks for the presence of the captcha JS.

The problem was both `should` and `should not` steps were asserting the same thing:

- `should`: be invisible
- `should not`: be visible

Which is essentially the same, and they should assert the opposite of each other. Now it's:

- `should`: be invisible
- `should not`: be invisible